package/kedro_viz/models/flowchart.py

```diff
@@ -806,7 +812,11 @@ def set_run_command(cls, _):
     @field_validator("preview")
     @classmethod
     def set_preview(cls, _):
-        if cls.data_node.is_preview_disabled() or not hasattr(cls.dataset, "preview"):
+        if (
+            cls.data_node.is_preview_disabled()
+            or not hasattr(cls.dataset, "preview")
+            or not cls.are_datasets_previewable
```

Comment:
What's the difference here between `cls.data_node.is_preview_disabled()` and `cls.are_datasets_previewable`? Semantically, it sounds very similar, so I think we should think carefully about how these flags are named so it's easy to understand what's happening for future team members. 

Also in terms of functionality, what does it mean for preview to be disabled on a node level vs class level?

Reply:
Agree with @merelcht. 

i am also thinking for clarity we could possibly have :- 
- is_preview_disabled on node level (and explain this in the docstring, that this disabled via node level happens when the user specifies 'preview=false' in the DataCatalog)
- is_all_previews_disabled on class level (and once again explain in docstring this is to disable previews for all nodes and can be done via CLI and UI)

Reply:
@merelcht As discussed with @rashidakanchwala the existing `is_preview_disabled` will rename to `is_preview_enabled` and new boolean variable `are_datasets_previewable` to `is_all_previews_enabled` .